main.go

```diff
@@ -607,6 +608,66 @@ func main() {
 					)
 				})
 			}
+
+			// Traces.
+			if cfg.traces.enabled && cfg.traces.readEndpoint != nil {
+				r.Group(func(r chi.Router) {
+					r.Use(authentication.WithTenantMiddlewares(pm.Middlewares))
+					r.Use(authentication.WithTenantHeader(cfg.traces.tenantHeader, tenantIDs))
+
+					// V2 query API, protected by RBAC
+					r.Mount("/api/traces/v1/{tenant}/api",
```

Comment:
After looking closely, I'm wondering why we're departing here from how the paths are mounted for metrics and traces. If I'm looking at the code correctly, we're mounting these paths, but we're handling everything on them via `/*`. Why aren't we handling only specific paths?

Reply:
Thanks for this comment.  I have restricted things.
Now the API handler only proxies GET for /api/traces, /api/dependencies, and /api/services.  The UI and static asset handlers still take /* but only proxy GET.

I am a bit confused about the code that I needed to do this.  I am doing `stripTenantPrefix("/api/traces/v1"()` then
```
		r.Get("/traces*", c.instrument.NewHandler(
			prometheus.Labels{"group": "tracesv1api", "handler": "api"},
			proxyRead))
```

I do not understand why it works when I use `r.Get("/traces*"...)`.  I had expected to be doing `r.Get("/api/traces*"...)